test/integration/nighthawk_test_server.py

```diff
@@ -63,6 +70,7 @@ def __init__(self, server_binary_path, config_template_path, server_ip, ip_versi
         server_binary_config_path_arg (str): Specify the name of the CLI argument the test server binary uses to accept a configuration path.
         parameters (dict): Supply to provide configuration template parameter replacement values.
         tag (str): Supply to get recognizeable output locations.
+        bootstrap_version_arg (str, optional): specify a bootstrap cli argument value for the test server binary.
```

Comment:
Same comment about making it an int and making the default value a numeric rather than a None.

We can still keep the meaning, e.g. the default could be zero and the code below can still do:
if self._bootstrap_version_arg:
  ...

Since zero evaluates to false. We can specify here in the comment what the use of zero value causes.

Reply:
Ah, I see, that makes sense. Following up.